modules/internal/StringCasts.chpl

```diff
@@ -71,24 +72,25 @@ module StringCasts {
 
   inline proc _cast(type t, x: string) where isIntegralType(t) {
     //TODO: switch to using qio's readf somehow
+    var str = x.strip();
     pragma "insert line file info"
-    extern proc c_string_to_int8_t  (x:c_string) : int(8);
+    extern proc c_string_to_int8_t  (str:c_string) : int(8);
```

Comment:
I don't think the names need to be updated.  These aren't calls, they're extern procedure declarations.